production_entry_app/production_entry_app/joint_production.py

```
@@ -0,0 +1,572 @@
+		"description": item.description,
+		"qty": flt(qty),
+		"transfer_qty": flt(qty),
+		"uom": item.stock_uom,
+		"stock_uom": item.stock_uom,
+		"conversion_factor": 1,
+		"s_warehouse": s_warehouse,
+		"t_warehouse": t_warehouse,
+	}
+
+
+def _get_rejection_warehouse(doc: Any) -> str:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `doc`

(ANN401)